deploy/flatten/che-workspace-crd-operator.v0.0.1.clusterserviceversion-v0.0.1.yaml

```diff
@@ -134,7 +134,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
-  name: che-workspace-crd-operator.v0.0.1
+  name: che-workspace-controller.v0.0.1
```

Comment:
Don't you also rename the file ?

Reply:
@davidfestal Could you suggest what is the right name for file and this field?
che-workspace-operator or che-workspace-controller

I guess che-workspace-operator

Reply:
Name is changed to 
`che-workspace-operator.v0.0.1`

File name now is `che-workspace-operator.v0.0.1.clusterserviceversion-v0.0.1.yaml`